Description: Point local_cleaners_dir to personal_cleaners_dir
Origin: Debian
Bug-Debian: http://bugs.debian.org/565359
Forwarded: not-needed

Index: bleachbit-0.7.4/bleachbit/Common.py
===================================================================
--- bleachbit-0.7.4.orig/bleachbit/Common.py	2010-01-24 21:09:13.798194540 +0100
+++ bleachbit-0.7.4/bleachbit/Common.py	2010-01-24 21:09:09.158190122 +0100
@@ -99,7 +99,7 @@
     system_cleaners_dir = '/usr/pkg/share/bleachbit/cleaners'
 
 # local cleaners directory (for running from source tree)
-local_cleaners_dir =  os.path.normpath(os.path.join(bleachbit_exe_path, '../cleaners'))
+local_cleaners_dir =  os.path.normpath(personal_cleaners_dir)
 
 # application icon
 __icons = ( '/usr/share/pixmaps/bleachbit.png', # Linux
